app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
```suggestion
        self.pople[name] = self
```
maybe it will be better

Reply:
no, here u must use class atribute not object of class